sound/soc/sof/trace.c

```diff
@@ -247,6 +251,8 @@ int snd_sof_trace_update_pos(struct snd_sof_dev *sdev,
 			     struct sof_ipc_dma_trace_posn *posn)
 {
 	if (sdev->dtrace_is_enabled && sdev->host_offset != posn->host_offset) {
+		snd_sof_trace_read_buf(sdev,
```

Comment:
I would also expect code here to remove the debugFS trace file.

Reply:
@lgirdwood Should we remove it after all trace_event function is ready? I.e. include the user space tool. Or can we let trace event and debugFS exist simultaneously for a while to make sure we didn't miss anything in trace event?

Reply:
lets remove in the same PR, but have a new PR for soft that has the updates to decode the trace data.